.github/workflows/build-ros2-ws-action.yml

```
@@ -0,0 +1,57 @@
+name: CI
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [master]  
+jobs:
+  build-and-test:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        run: git clone -b feature/fix-using-grpc https://github.com/ichiro-its/aruku.git
```

Please revert this to use the Checkout action. My previous suggestion only applies to external repositories, not to the current repository. You may also need to keep the `path` input, otherwise, it will be cloned into the current working directory, which would cause the other packages to be cloned inside the Aruku package instead of having the Aruku package at the same level as the other packages.

```suggestion
      - name: Checkout
        uses: actions/checkout@v4.1.1
        with:
          path: aruku
```